src/mixins/legal-api-mixin.ts

```diff
@@ -64,6 +64,21 @@ export default class LegalApiMixin extends Mixins(FilingTemplateMixin) {
       })
   }
 
+  /**
+   * Deletes a filing by its id.
+   * @returns a promise to delete the filing or return a failure.
+   */
+  async deleteFilingById (id: number): Promise<any> {
+    const url = `businesses/${this.getBusinessId}/filings/${id}`
+
+    return axios.delete(url)
+      .then().catch(error => {
```

Comment:
ok cool, will remove!

Reply:
Or perhaps this would make more sense here?
```
await axios.delete(url)
   .catch(/* whatever */)
return
```

It depends on how you want to handle the async / promise.